src/SelectMenuContext.js

```diff
@@ -0,0 +1,7 @@
+import React, {createContext} from 'react'
+
+export const MenuContext = createContext()
+
+export const ContextProvider = ({value, children}) => {
```

Comment:
Can this be named something with `Menu` in it? When you import {ContextProvider} in another file it's difficult to understand what it's for.

That being said, is there any specific reason to create a new component here? Can we just export MenuContext and be done with it?